src/json/util.h

```diff
@@ -62,6 +62,7 @@ typedef enum {
     JSONUTIL_CANNOT_INSERT_MEMBER_INTO_NON_OBJECT_VALUE,
     JSONUTIL_INVALID_RDB_FORMAT,
     JSONUTIL_DOLLAR_CANNOT_APPLY_TO_NON_ROOT,
+    JSONUTIL_MEMORY_ERROR,
```

Comment:
Also need to modify jsonutil_code_to_message in util.cc, to add a case statement for the enum. That will resolve the unit test failure.